python/cudf/cudf/core/column/datetime.py

```diff
@@ -459,19 +416,72 @@ def as_timedelta_column(self, dtype: np.dtype) -> None:  # type: ignore[override
             f"cannot astype a datetimelike from {self.dtype} to {dtype}"
         )
 
+    @functools.cached_property
+    def _strftime_names(self) -> plc.Column:
+        """Strftime names for %A, %a, %B, %b"""
+        return plc.Column.from_iterable_of_py(
+            [
+                nl_langinfo(loc)
+                for loc in (
+                    locale.AM_STR,
```

Comment:
In case you wanted to reduce typing even further...
```suggestion
                nl_langinfo(getattr(locale, loc))
                for loc in (
                    "AM_STR",
```
(and also changing the rest of the list of course).

Reply:
I don't want Cursor to be upset that it didn't simplify this enough :)